bin/pfcmd.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -T
+#!/usr/bin/perl
```

Comment:
Indeed, and IMO running pfcmd with that flag doesn't represent the environment that the app runs in since (correct me if I'm wrong) but the other processes run without the tainting check.

As you said, we will revisit this, we can also brainstorm on the validity of these checks